modules/management-console-agent/src/main/java/org/apache/ignite/agent/processor/ManagementConsoleTopicProcessor.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.agent.processor;
+
+import java.util.UUID;
+import org.apache.ignite.agent.dto.NodeConfiguration;
+import org.apache.ignite.agent.dto.tracing.SpanList;
+import org.apache.ignite.agent.ws.RetryableSender;
+import org.apache.ignite.internal.GridKernalContext;
+import org.apache.ignite.internal.processors.GridProcessorAdapter;
+import org.apache.ignite.internal.visor.event.VisorGridEvent;
+import org.apache.ignite.lang.IgniteBiPredicate;
+
+import static org.apache.ignite.agent.ManagementConsoleAgent.TOPIC_MANAGEMENT_CONSOLE;
+import static org.apache.ignite.agent.StompDestinationsUtils.buildClusterNodeConfigurationDest;
+import static org.apache.ignite.agent.StompDestinationsUtils.buildEventsDest;
+import static org.apache.ignite.agent.StompDestinationsUtils.buildSaveSpanDest;
+import static org.apache.ignite.agent.utils.AgentUtils.quiteStop;
+
+/**
+ * Management console topic processor.
+ */
+public class ManagementConsoleTopicProcessor extends GridProcessorAdapter {
```

Comment:
May be ManagementConsoleTransitionProcessor or ManagementConsoleExporterProcessor or ManagementConsoleMessagingProcessor will be better?